.flake8

```diff
@@ -1,5 +1,5 @@
 [flake8]
 # E722 - do not use bare 'except'
-ignore = E722
+ignore = E722, W504
```

Comment:
can you please explain in the comments, why W504 added.

Reply:
comment added explaining the reason for W504.